CHANGELOG.md

```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
 
 ## [v4.14.1]
 
+### Ruleset
+
+#### Changed
+
+- Reworked SCA Policy for Microsoft Windows 10 Enterprise. ([#31449](https://github.com/wazuh/wazuh/pull/31449))
+- Fixed bug in Windows SCA. ([#31389](https://github.com/wazuh/wazuh/pull/31349))
```

Comment:
- Fixed bug in Windows SCA. ([#31349](https://github.com/wazuh/wazuh/pull/31349))?